packages/tasks/src/snippets/python.ts

```diff
@@ -16,9 +16,24 @@ export const snippetConversational = (
 	}
 ): InferenceSnippet[] => {
 	const streaming = opts?.streaming ?? true;
-	const messages: ChatCompletionInputMessage[] = opts?.messages ?? [
-		{ role: "user", content: "What is the capital of France?" },
-	];
+	const exampleMessages: ChatCompletionInputMessage[] =
+		model.pipeline_tag === "text-generation"
+			? [{ role: "user", content: "What is the capital of France?" }]
+			: [
+					{
+						role: "user",
+						content: [
+							{
+								type: "image_url",
+								image_url: {
+									url: "https://cdn.britannica.com/61/93061-050-99147DCE/Statue-of-Liberty-Island-New-York-Bay.jpg",
+								},
+							},
+							{ type: "text", text: "Describe this image in one sentence." },
+						],
+					},
+			  ];
```

Comment:
Feels like these example messages should be in [inputs.ts](https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/snippets/inputs.ts). Can be done in a later PR.

Reply:
https://github.com/huggingface/huggingface.js/pull/989